Eastern-Research-Group/mywaterway
# Bugfix/446 fix saved data other pages

## Related Issues:
* [HMW-446](https://jira.epa.gov/browse/HMW-446)

## Main Changes:
* Fixed issues with saving from the monitoring report, plan summary, state, tribe, and waterbody report pages.

## Steps To Test:
1. Navigate test saving from pages other than the community page
2. Verify that web maps in AGO match what you expect
